.github/workflows/test.yml

```
@@ -2,7 +2,7 @@ name: test
 
 on:
   push: # Run on pushes to the default branch
-    branches: [main]
+    branches: [main,enable-ci-dbt-async-1.6-1.9-af3]
```

```suggestion
    branches: [main]
```
Remove before merging